hw09-aop/src/main/java/ru/otus/merets/HW09Main.java

```diff
@@ -0,0 +1,22 @@
+package ru.otus.merets;
+
+import ru.otus.merets.blackbox.BoxImpl;
+import ru.otus.merets.blackbox.BoxInterface;
+import ru.otus.merets.exception.CreateWatcherException;
+import ru.otus.merets.watcher.Watcher;
+
+public class HW09Main {
+
+    public static void main(String[] args) throws CreateWatcherException {
+
+        Watcher watcher = new Watcher<BoxInterface>();
+        BoxInterface myClass = (BoxInterface) watcher.createMyClass(BoxImpl.class);
```

Comment:
```suggestion
        Box box = (Box) watcher.createMyClass(BoxImpl.class);
```

Слово "Interface" не принято писать в именах интерфейсов.
Лучше давать осмысленные названия переменным.